format_checker/pr_checker.py

```diff
@@ -51,6 +51,7 @@
         "Skipped",
         "MovedOrRenamed",
         "Claimed",
+        "MovedToGradle"
```

Comment:
Should there be a comma at the end of the line, as the previous `Claimed` seems to have had a comma at the end?